test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/CompositeModelBinderTest.cs

```diff
@@ -65,6 +65,7 @@ public class CompositeModelBinderTest
             var bindingContext = new ModelBindingContext
             {
                 FallbackToEmptyPrefix = true,
+                IsTopLevelObject = true,
```

Comment:
Why was this addition or the one at [line 497](https://github.com/aspnet/Mvc/pull/3049/files#diff-2692e9de2424d15a779aa403c7c6c95aR497) necessary? If needed, why were the tests passing before?